spec/integration/halBoard.json

```diff
@@ -65,7 +67,8 @@
 									"href": "/test/api/card/301/block",
 									"method": "PUT"
 								}
-							}
+							},
+							"_version": 1
```

Comment:
It won't. You can't request the parent resource as a version and get something other than that from the child. Even if the child didn't have version X, the way versions are applied, the child will have the same "effective" version despite there being no difference between its last version and the parent's.